components/mosart/src/riverroute/RtmMod.F90

```diff
@@ -2141,9 +2148,9 @@ subroutine Rtmrun(rstwr,nlend,rdate)
     integer,parameter :: bVelo_upChnlNo   = 58 ! Total number of channels with upward flow velocities (dimensionless).
 
     ! Sediment TERMS (rates kg/s or storage kg)
-    integer,parameter :: br_ehexch = 70 ! exchanging fluxes between channel and enviroments
-    integer,parameter :: br_etexch = 71 ! exchanging fluxes between channel and enviroments
-    integer,parameter :: br_erexch = 72 ! exchanging fluxes between channel and enviroments
+    integer,parameter :: br_ehexch = 70 ! exchanging fluxes between channel and environments
```

Comment:
I only have one more comment. What environments here refer to? Land?

Reply:
Anything except for rivers, i.e., land, atmosphere, etc. I keep it a bit vague here for potential future extensions.